src/screens/Home/ReceiveTokensWarning.tsx

```diff
@@ -99,7 +99,7 @@ const ReceiveTokensWarning: FC<IReceiveTokensWarning> = ({ theme }) => {
       if (!chainRates || !gasInfo?.gasPrice?.fast) return null;
       return calculateDeploymentFee(chain, chainRates, fiatCurrency, gasInfo);
     },
-    [chainRatesPerChain, chainRatesPerChain, fiatCurrency],
+    [chainRatesPerChain, chainRatesPerChain, fiatCurrency, gasInfoPerChain],
```

Comment:
Was the issue here that when the gas information became available, we didn't refresh the UI?

Reply:
Yes